pkg/operation/operation.go

```diff
@@ -434,27 +434,27 @@ func (o *Operation) WantsPlutono() bool {
 
 // ComputeKubeAPIServerHost computes the host with a TLS certificate from a trusted origin for KubeAPIServer.
 func (o *Operation) ComputeKubeAPIServerHost() string {
-	return o.ComputeIngressHost(common.KubeAPIServerPrefix)
+	return o.ComputeIngressHost("api")
```

Comment:
Isn't it better to keep the constants for `api` and similar for others?

Reply:
They were only used here, so there is no need to keep them with just a single usage.